Nef_3/include/CGAL/Nef_3/K3_tree.h

```diff
@@ -381,45 +365,42 @@ typedef Smaller_than<
     return s;
   }
 
-  template<typename Depth>
```

Comment:
I am puzzled to see `Depth`  as template parameter.  Wouldn't that need a constant known at compile time. @lrineau @mglisse

Reply:
Its always an int, its specialised by a parameter `build_k3tree(O, O.size()+v_end, 0)`

Reply:
But isn't `O.size()+v_end`  only known at runtime?

Reply:
No, it's the final parameter, i.e the `0` which is a literal int.

Reply:
I misunderstood. I thought we have to do with for example `Depth-1`  being used in a recursion, but here it is just an integral type in the parameter list. So I agree with this change.